playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -88,29 +127,9 @@ cd ../02.builders
 
 # Run terraform commands
 terraform init -backend-config="bucket=$STATE_BUCKET"
-terraform apply -var="project_id=$(gcloud config get-value project)" -var="state_bucket=$STATE_BUCKET"
+terraform apply -var="project_id=$(gcloud config get-value project)" -var="state_bucket=$STATE_BUCKET" -var-file="$BEAM_ROOT/playground/terraform/environment/$ENVIRONMENT_NAME/common.tfvars"
```

Comment:
Do we need state_bucket var in terraform apply command line?

Reply:
No need for separate var, its provided via commons varfile